### PR TITLE
Support iOS / Android

### PR DESCRIPTION
**Pull request by longfin**
It should be merged after https://github.com/planetarium/lib9c/pull/1714

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,160 +1,97 @@
...
-            echo -n "internal"
-              ;;
-
-            refs/heads/previewnet)
-              echo -n "previewnet"
-              ;;
+          key: Library-${{ matrix.projectPath }}-ubuntu-${{ matrix.targetPlatform }}
+          restore-keys: |
+            Library-${{ matrix.projectPath }}-ubuntu-
+            Library-
```
[the remark this answers is not in the document]
Comment (longfin) [reply .github/workflows/build.yml:49]: In general, caching often be required with commit hash to indicate specific points. 😕

File: .gitignore [hunk 2]
```diff
@@ -59,7 +59,6 @@ sysinfo.txt
 nekoyume/compose/results
 
 # Pseudo commandline options.
-/nekoyume/Assets/StreamingAssets/clo.json*
```
Comment (longfin) [inline .gitignore:62]: My bad. I'll restore it.

File: .github/workflows/build.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Atralupus) [reply .github/workflows/build.yml:49]: After reviewing this [docs](https://game.ci/docs/github/builder#complete-example), I copied it.
Based on my understanding, it is acceptable to use the same packages and cache during the Unity build, as the necessary components for each platform are automatically deleted before starting the build.

[the remark this answers is not in the document]
Comment (longfin) [reply .github/workflows/build.yml:49]: Thanks for the explanation. 🆗

File: .gitignore [hunk 3]
```diff
@@ -59,7 +59,6 @@ sysinfo.txt
 nekoyume/compose/results
 
 # Pseudo commandline options.
```
Comment (longfin) [inline .gitignore:61]:
```suggestion
# Pseudo commandline options.
/nekoyume/Assets/StreamingAssets/clo.json*
```

File: .gitignore (hunk 2 above)
Comment (longfin) [reply .gitignore]: Added. https://github.com/planetarium/NineChronicles/pull/2351/commits/d1b478043bc956435c2e31b7176b8380332b312c

**Pull request merged**